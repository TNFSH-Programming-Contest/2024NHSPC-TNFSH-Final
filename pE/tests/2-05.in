1 9912978 7115237 8235356
1379652 822877 822877
47
