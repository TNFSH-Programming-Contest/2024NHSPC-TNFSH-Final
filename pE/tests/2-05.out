588026151239778
